src/components/command-bar/commands/chat/index.tsx

```diff
@@ -7,33 +7,39 @@ import { IconArrowLeft24 } from '@hashicorp/flight-icons/svg-react/arrow-left-24
 import { CommandBarCommand } from 'components/command-bar/types'
 import ChatBox from 'components/chatbox/chatbox'
 import { useCommandBar } from 'components/command-bar'
+import Text from 'components/text'
 
 import s from './chat.module.css'
 
-// An interactive Icon w/ "back" functionality
+// A button with an icon and text w/ "back" functionality
+// TODO this is hacky?
 const Icon = () => {
```

Comment:
I know this should be able to be something else, but I don't know how to change it successfully.